packages/commonwealth/scripts/create-peer-id.ts

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
```

Comment:
Does this have to be a JS file or could it be converted to TS given that all of our non-bash scripts are in TS?

Reply:
Removed the hashbang and tested, looks like this works fine with just TS.